test/verify/check-networking-bond

```diff
@@ -193,10 +198,54 @@ class TestNetworking(NetworkCase):
         b.click("#networking-interfaces tr[data-interface='tbond'] td:first-child")
         b.wait_visible("#network-interface")
         b.wait_present("#network-interface-slaves tr[data-interface='%s']" % iface)
-        b.wait_in_text("#network-interface .panel:contains('tbond')", ip)
+        # We want that to be true, but this only works for identical MACs (https://bugzilla.redhat.com/show_bug.cgi?id=1548265)
```

Comment:
This is the whole point of the test, no?  To ensure that bonding a active interface does not change the IP address of that interface.